src/core_atmosphere/Registry.xml

```diff
@@ -336,6 +336,12 @@
                      possible_values="Non-negative real values"/>
 	</nml_record>
 
+        <nml_record name="assimilation" in_defaults="true">
```

Comment:
If MPAS-A is supported to the general community in JEDI, then I think it makes sense to keep the `assimilation` namelist record in the default namelist; otherwise, I'd propose to set `in_defaults="false"` so that the `&assimilation` namelist record will still be valid, but it just won't appear by default in the `namelist.atmosphere` file that is auto-generated at build time.

Reply:
That makes sense.  I admit that I did not know what that entry meant.  Fixing it now.

Reply:
No worries -- our documentation is non-existent in many cases!